### ISSUE TITLE
Add addons-linter to Travis-CI or package.json

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blok#35]
Current output:

```sh
$ ./node_modules/.bin/addons-linter web-ext-artifacts/*.zip -o text
Validation Summary:

errors          0
notices         0
warnings        0
```

**Comment by groovecoder** [comment]
Looks like https://github.com/mozilla/blok/pull/39/files includes addons-linter in the package.json, but there's not a corresponding script for it? Should we add that to the same PR?

**Comment by pdehaan** [comment]
Yeah, sorry, I think I got a bit sloppy on the weekend with leaving that in das package.json.
I wasn't sure how to call the `addons-linter` since the web-ext-artifacts directory includes multiple XPI/ZIP files. We may want to just do an `rm -rf web-ext-artifacts/*` and then rebuild the add-on and only lint the latest version (as opposed to constantly re-linting old add-ons).

We can probably merge PR #39 as-is and I'll see about cleaning up Travis addons-linter usage in a separate PR (since it's nice to know the add-on is error free and compiles cleanly).

**Comment by pdehaan** [comment]
Initial stab at this in #44.
Once we land the **standard** PR, I can clean it up by converting the .travis.yml stuff to devDependencies.
